projects/addon-editor/components/toolbar-tools/highlight-color/highlight-color.component.ts

```diff
@@ -17,7 +21,7 @@ import {distinctUntilChanged, map} from 'rxjs/operators';
 export class TuiHighlightColorComponent {
     @Input()
     @tuiDefaultProp()
-    colors: ReadonlyMap<string, string> = defaultEditorColors;
+    colors: ReadonlyMap<string, string> = this.defaultOptions.colors;
 
     readonly backgroundColor$ = this.editor.stateChange$.pipe(
         map(() => this.editor.getBackgroundColor() || EDITOR_BLANK_COLOR),
```

Comment:
what do you think about move `EDITOR_BLANK_COLOR` to options?

Reply:
✅